
	{% extends './layout/sub_layout.html' %}

	{% block head%}
		<script type="text/javascript">

		</script>
	{% endblock %}


	<!-- 컨텐츠 컨테이너 시작-->
	{% block ContentContainer %}
	<div class="container pfarticipation">
		<div class="subVisual inquiry">
			<div class="mv_content">
				<div class="text_area">
					<span>Tutorial</span>
				</div>
			</div>
		</div>
		
		<!-- 제목 시작 -->
		{% block titleArea %}
			<div class="title_area">
				<span>AIVE Tutorial</span>
			</div>
		{% endblock %}
		<!-- //제목 종료 -->
		
		
		<!-- topmenu start -->
		{% block mainContent%}
		<div class="board_area" >
			<!--슬라이드2 시작-->
			<div class="tutorial_img01">
				<img src="/static/img/tutorial/img_tutorial_top.png" />
			</div>
			<p span class="title_num2">(1) AIVE can be used to evaluate :</p>
			<div class="tutorial_expl02">
				<dt>
					A. Protein structure prediction from viral amino acid sequences<p></p>
					B. Analysis of repeated polarity in viral amino acid sequences (PCS)<p></p>
					C. Rate of change of mutations for viral genome sequences (MR)<p></p>
					D. Scoring of amino acid changes based on amino acid properties (BPES)<p></p>
					E. Intergrative mathematical analysis of protein structure prediction (amino acid clustering), PCS, MR, and, BPES
				</dt> 
			</div>
			<p span class="title_num2">(2) The input range of analysis for AIVE is as follows :</p>
			<div class="tutorial_expl02">
				<dt>
					A. Basically provided SARS-CoV-2 VOCs
				</dt> 
				<dl>
					<dd>
						- Monomer (single amino acid sequence)<p></p>
						- Multimer (two or more amino acid sequences, when analyzing docking score, only two sequences are possible)
					</dd>
				</dl>
				<dt>
					<p></p>
					B. Analysis of amino acid sequences written by the user
				</dt> 
				<dl>
					<dd>
						- Monomer (single amino acid sequence)<p></p>
						- Multimer (two or more amino acid sequences, when analyzing docking score, only two sequences are possible)<p></p>
						- PDB and JSON files that users possess can be directly submitted
					</dd>
				</dl>
		    </div>
			<div class="tutorial_step">
				Sign up & Login
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/tutorial_signup.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					AIVE is used by creating a personal account for the ease of comparing analysis results.<p></p>
				</dt> 
				<dl>
					<dd>
						① Click on the "Sign Up" option to proceed with the account creation.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/tutorial_login.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
				</dt> 
				<dl>
					<dd>
						① Enter the username and password. To use the "Forgot password?" feature, enter your email and proceed with account creation.<p></p>
						②, ③ Access the "Login" section, enter the account information, and log in.
					</dd>
				</dl>
			</div>

			<div class="tutorial_step">
				Demo
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_demo.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					AIVE provides a demo feature that demonstrates the analysis of SARS-CoV-2 variants of concern (VOCs).<p></p>
				</dt> 
				<dl>
					<dd>
						① Clicking on the Demo button will automatically select the RBM [S:437-508] region of the SARS-CoV-2 BA.5 variant.<p></p>
						② To predict and analyze the structure of the sequence generated by the Demo function, click the "Server Prediction" button to submit the task.<p></p>
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				Predict VOC structure
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_voc.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					In AIVE, users can personally select and analyze the VOC and domains of SARS-CoV-2.<p></p>
				</dt> 
				<dl>
					<dd>
						① Write the Project name for task categorization.<p></p>
						② Select SARS-CoV-2 in the Target Virus section to retrieve information on Coronaviruses.<p></p>
						③, ④, ⑤ Choose the VOC and regions you want to analyze to retrieve the corresponding Amino Acid sequence information.
					</dd>
				</dl>
			</div>
			<p class="title_when2">Submit</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_submit.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
				</dt> 
				<dl>
					<dd>
						⑥ Click the "Server Prediction" button to start the prediction and analysis of the Project
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				List
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_list.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					In the "List" section, users can check the list of submitted Projects and their progress.
				</dt> 
				<dl>
					<dd>
						① Click the "Report menu” and then select "List" from the submenu to check Projects.<p></p>
						② You can check the analysis results by clicking on the "Result info" section for completed tasks.
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				Generate SARS-CoV-2 mutated sequence
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_generate.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					Users can not only access information about VOCs but also generate mutations for analysis.
				</dt> 
				<dl>
					<dd>
						① Specify a Project name to categorize the submitted task<p></p>
						②, ③, ④, ⑤ Choose Wuhan-HU-1 sequence of SARS-CoV-2 and retrieve the sequence of the region you want to verify.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_generate2.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
				</dt> 
				<dl>
					<dd>
						⑥ In Alignment, click on the positions in the sequence of Wuhan-HU-1 to select the mutated Amino Acid.<p></p>
						⑦ Additionally, choose which codon to mutate into from the selected Amino Acid. This function is used to assess the impact of mutations at the codon level.<p></p>
						⑧ The sequence with the selected mutations is displayed.<p></p>
						⑨ Submit the task to proceed with the structural prediction and analysis of the mutated sequence.
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				User sequence - Monomer
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_monomer.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					In AIVE, users can predict the structure of sequences, not limited to SARS-CoV-2. Let's take a look at monomer structure prediction
				</dt> 
				<dl>
					<dd>
						① Select "All viruses" in the Target Virus section.<p></p>
						② Enter the Amino Acid sequence you want to check in the "Input virus Sequence" box.<p></p>
						③ Alternatively, you can upload a fasta file instead of entering it directly.<p></p>
						④ When the upload window appears, select the fasta file you want to check to pull up the sequence.
					</dd>
				</dl>
			</div>
			<p class="title_when2">User sequence</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_userseq.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
				</dt> 
				<dl>
					<dd>
						⑥ The Amino Acid sequence recorded in the fasta file you uploaded is displayed in “Alignment”.<p></p>
						⑦ Click on the positions in the displayed sequence where you want to introduce mutations and select the mutated Amino Acid.<p></p>
						⑧ Additionally, choose which codon to mutate into from the selected Amino Acid. This function is used to analyze the impact of mutations at the codon level.<p></p>
						⑨ The sequence with the selected mutations is displayed.<p></p>
						⑩ Submit the task to proceed with the structural prediction and analysis of the mutated sequence.
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				User sequence - Multimer
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_multimer.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					Let's look at the case of predicting a Protein Complex structure:
				</dt> 
				<dl>
					<dd>
						① Click the +Add button to create as many sequence input boxes as there are chains in the protein complex you want to predict.<p></p>
						② Enter the Amino Acid sequence of each chain in the generated “Input virus Sequence” boxes.<p></p>
						③ Alternatively, a fasta file can be uploaded without entering the sequence directly<p></p>
						④ When the upload window appears, select the fasta file to retrieve the sequence.<p></p>
						⑤ The sequences of each chain, as stored in the fasta file, are inputted.
					</dd>
				</dl>
			</div>
			<p class="title_when2">User sequence - Multimer</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_userseq2.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
				</dt> 
				<dl>
					<dd>
						⑥ Click on the positions in the displayed sequences where you want to introduce mutations and select the mutated Amino Acid.<p></p>
						⑦ Additionally, choose which codon to mutate into from the selected Amino Acid. This function is used to analyze the impact of mutations at the codon level.<p></p>
						⑧ The sequence with the selected mutations is displayed.<p></p>
						⑨ Submit the task to proceed with the structural prediction and analysis of the mutated sequence.
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				Result report page - Structure
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_result_structure.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					From the prediction and analysis results of SARS-CoV-2, the 3D structure prediction results can be accessed.
				</dt> 
				<dl>
					<dd>
						① You can select and view the predicted 3D structures by choosing from the five available options.<p></p>
						② Use the "Download all file" button to download the result files of the predicted structures to your device.<p></p>
						③ You can visualize the predicted 3D structure for inspection and comparison with SARS-CoV-2 Wuhan-HU-1. ④, ⑤ Clicking on the highlighted regions in positions allows you to inspect them.<p></p>
						④ Predicted aligned error (PAE) is a value that estimates the difference between the relative locations of two residues of the model and the real model. A low PAE value indicates that the accuracy of the relative location of the two residues is high.<p></p>
						• The color at (x, y) indicates AlphaFold’s expected position error at residue x if the predicted and true structures were aligned on residue y.<p></p>
						• If the PAE is generally low for residue pairs x, y from two different domains, it indicates that AlphaFold predicts well-defined relative positions and orientations for them. (Explanation from AlphaFold FAQ)<p></p>
						⑤ Predicted LDDT(pLDDT) is a value that estimates the reliability of the model. It estimates how well the actual model residue and predicted model residue match. At the same time, it indicates how well the protein structure folds in the corresponding location. A low pLDDT value indicates that the reliability of the corresponding position is low and that it possesses a disordered structure.<p></p>
							</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				Result report page - APESS
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_result_apess.png" />
				</div>
			</div>
			<p class="title_when2">Result report page – APESS subscore</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_result_apess_score.png" />
				</div>
			</div>
			<p span class="title_num2">① SCPS</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_scps.png" />
				</div>
			</div>
			<p span class="title_num2">② PCS</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_pcs.png" />
				</div>
			</div>
			<p span class="title_num2">③ BPES</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_bpes.png" />
				</div>
			</div>
			<p span class="title_num2">④ MR</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_mr.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					For the amino acid sequence entered by the user, the AIVE system provides a total of 6 evaluation charts.
				</dt> 
				<dl>
					<dd>
						① AIVE predicts protein structures due to mutations in each gene for coronavirus lineages or sub-lineages. From the predicted result, it carries out grouping of amino acids (components of 3D protein structure) utilizing K-means clustering to report SCPS results.<p></p>
						② through repeated pattern analysis of polar amino acids in amino acid sequences, AIVE reports PCS results.<p></p>
						③ AIVE figures out amino acid properties to measure BPES through measurement of changes in biochemical properties of amino acid.<p></p>
						④ AIVE calculates MR through rate of change for nucleotide frequencies due to mutations. 
					</dd>
				</dl>
			</div>
			<p class="title_when2">Result report page – APESS & distribution</p>
			<p span class="title_num2">⑤ APESS</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_apess.png" />
				</div>
			</div>
			<p span class="title_num2">⑥ APESS distribution graph</p>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_apess_dist.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					From the prediction and analysis results of SARS-CoV-2, the 3D structure prediction results can be accessed.
				</dt> 
				<dl>
					<dd>
						⑤ APESS, the result of comprehensive mathematical model (SCP*PCS*MR*BPES) of measured analysis results is provided.<p></p>
						⑥ We calculated the APESS score for a dataset of 7 million samples collected from GISAID. We then applied a Gaussian Mixture Model (GMM) to identify and represent it with 4 components. This allows us to provide the likelihood of which component the sequence is most likely to belong to from the APESS score.
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				Result report page - Polarity
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_polarity.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					Amino acid polarity affects protein structure and stability. As a result, the amino acid polarity due to mutation of the amino acid sequence input by the user can be observed. We found repeated polarity patterns in the coronavirus and observed changes in the properties of amino acid sequence polarity due to mutation.<p></p>
					Therefore, we provide visualization and table view of polarity pattern changes to the user.

				</dt> 
				<dl>
					<dd>
						① amino acid sequence<p></p>
						② 4 polarity characteristics<p></p>
						③ 5 amino acid properties<p></p>
						④ Mutated positions are indicated in red.
					</dd>
				</dl>
			</div>
			<div class="tutorial_step">
				Result report page – All viruses
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_allvirus.png" />
				</div>
			</div>
			<div class="tutorial_step">
				Result report page –Compare
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_compare.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					Structures predicted by “All viruses” can be compared with other structures using the compare feature.
				</dt> 
				<dl>
					<dd>
						① Click the "Compare" button to load the list of other tasks submitted by the user.<p></p>
						② Select the task you want to compare and click on it.<p></p>
						③ Click the "Confirm" button to navigate to the comparison page.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_compare2.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
					Through the "Compare" feature, you can compare two predicted structures in the list:
				</dt> 
				<dl>
					<dd>
						④ You can visually inspect the PAE (Predicted Alignment Error) of the two structures using a plot.<p></p>
						⑤ You can compare the pLDDT values of the two structures.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img03">
					<img src="/static/img/tutorial/t_compare3.png" />
				</div>
			</div>
			<div class="tutorial_expl03">
				<dt>
				</dt> 
				<dl>
					<dd>
						⑥ The sequence and polar structures of the two structures can be compared and analyzed.
					</dd>
				</dl>
			</div>
		{% endblock %}
	</div>
	{% endblock %}

	


